.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "stage": 0
+  "presets": [
+    "es2015",
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
+    "add-module-exports"
+  ]
```

Comment:
Upgrade to babel 6 presets.  Since we have karma running from node, we need common js interop.  This plugin adds the module exports to each file to achieve that.